pages/item.server.js

```diff
@@ -4,7 +4,3 @@ export default function Item({ router }) {
   const { id } = router.query
   return <ItemPage id={id} />
 }
-
```

Comment:
We need this to force item page is rendering with edge runtime, otherwise it will be static opt and throw client error now. let's keep it for now until we have a better solution for `router` usage in rsc